src/schema/schema-generator.ts

```diff
@@ -360,7 +360,7 @@ export abstract class SchemaGenerator {
         description: handler.description,
         deprecationReason: handler.deprecationReason,
         complexity: handler.complexity,
-      };
+      } as GraphQLFieldConfig<any, any>;
```

Comment:
What is the reason of that? Declaration merging isn't working?
https://github.com/slicknode/graphql-query-complexity/blob/157285004e22eed3ca1ab1b43226d671f0c3affb/src/QueryComplexity.ts#L32-L36

Reply:
Getting this error if I remove this 
![image](https://user-images.githubusercontent.com/4750072/45242825-1f8cc280-b30f-11e8-8c85-36710aeb973b.png)